radix-engine-tests/benches/costing.rs

```diff
@@ -259,7 +281,41 @@ fn bench_sha256(c: &mut Criterion) {
             >= 99_000_000
     );
 
-    c.bench_function("costing::sha256", |b| {
+    c.bench_function("costing::scrypto_sbor_decode", |b| {
+        b.iter(|| ledger.execute_manifest(manifest.clone(), []))
+    });
+}
+
+// Usage: cargo bench --bench costing -- scrypto_malloc
+fn bench_scrypto_malloc(c: &mut Criterion) {
+    let mut ledger = LedgerSimulatorBuilder::new().build();
+    let package_address = ledger.publish_package_simple(PackageLoader::get("costing_malloc"));
+
+    let manifest = ManifestBuilder::new()
+        .lock_fee_from_faucet()
+        .then(|mut builder| {
+            for _ in 0..100 {
+                builder = builder.call_function(
```

Comment:
I imagine most of the cost units are actually on reading the package substate and instantiating the wasm, but I think it's fine to have this test 👍

Reply:
Was curious about it to :) 
<details>
<summary>Here is a sorted execution summary</summary>

```
Execution Cost Breakdown                        99999802,    100.0%
RunWasmCode::Test_f                             96019412,     96.0%
OpenSubstate::GlobalPackage                      1736073,      1.7%
ReadSubstate                                      759207,      0.8%
PrepareWasmCode                                   713194,      0.7%
ValidateTxPayload                                 255000,      0.3%
OpenSubstate::GlobalFungibleResourceManager       121872,      0.1%
OpenSubstate::InternalFungibleVault                90202,      0.1%
CheckReference                                     80024,      0.1%
RunNativeCode::lock_fee                            45243,      0.0%
OpenSubstate::GlobalGenericComponent               43690,      0.0%
OpenSubstate::InternalKeyValueStore                40536,      0.0%
RunWasmCode::Faucet_lock_fee                       27442,      0.0%
RunNativeCode::get_amount_FungibleVault            14451,      0.0%
CloseSubstate                                      11610,      0.0%
DropNode                                            9754,      0.0%
CreateNode                                          7868,      0.0%
OpenSubstate::InternalGenericComponent              7742,      0.0%
VerifyTxSignatures                                  7000,      0.0%
WriteSubstate                                       3900,      0.0%
SwitchStack                                         1000,      0.0%
QueryActor                                          1000,      0.0%
AllocateNodeId                                       873,      0.0%
BeforeInvoke                                         802,      0.0%
SetCallFrameData                                     606,      0.0%
EmitEvent                                            556,      0.0%
LockFee                                              500,      0.0%
PinNode                                              108,      0.0%
AfterInvoke                                           82,      0.0%
MarkSubstateAsTransient                               55,      0.0%
```
</details>

Test_f still consumes 96% of CPU.